<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatDlg</class>
 <widget class="QDialog" name="GroupChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="group_vatar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>avatar</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_min">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="group_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>groupname</string>
   </property>
  </widget>
  <widget class="GifTextEdit" name="group_announcement">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>86</y>
     <width>412</width>
     <height>218</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="toolbar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>334</y>
     <width>420</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="GifTextEdit" name="messageEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>412</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>467</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;Ctrl+Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>467</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;Ctrl+Enter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QListView" name="group_memberlist">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>86</y>
     <width>158</width>
     <height>407</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/transparent.png);</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QFrame" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>493</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/chatDlg_Bg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="face">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>304</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/faceButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showChatLog">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>304</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;Ctrl+Enter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="GifTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>411</width>
     <height>481</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'åææ¥·ä½'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>btn_min</zorder>
  <zorder>btn_close</zorder>
  <zorder>group_name</zorder>
  <zorder>group_announcement</zorder>
  <zorder>toolbar</zorder>
  <zorder>messageEdit</zorder>
  <zorder>closeButton</zorder>
  <zorder>sendButton</zorder>
  <zorder>group_memberlist</zorder>
  <zorder>group_vatar</zorder>
  <zorder>face</zorder>
  <zorder>showChatLog</zorder>
  <zorder>log</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GifTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">giftextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
